ui-events/src/pointer/buttons.rs

```diff
@@ -6,43 +6,77 @@
 #[repr(u32)]
 pub enum PointerButton {
```

Comment:
It might be nice to document why B7-31 exist (to match potential extra buttons on the mouse)

Reply:
Added some documentation of this.